src/components/html-text-input.js

```diff
@@ -83,9 +100,9 @@ export class HTMLInputView extends React.Component<PropsType, StateType> {
 	render() {
 		return (
 			<KeyboardAvoidingView style={ styles.container } parentHeight={ this.props.parentHeight }>
-				<ScrollView
-					style={ { flex: 1 } }
-					keyboardDismissMode="interactive" >
+				<View
+					{ ...( this.isIOS ? { ...this.panResponder.panHandlers } : {} ) }
+					style={ { flex: 1 } } >
```

Comment:
Do you think it's a good idea to remove this extra wrapper view and add the panHandlers directly to `KeyboardAvoidingView`?
It seems to work well but I wonder if there's a benefit on keeping this wrapper.

Reply:
there's no benefit currently so I removed